heroku.yml

```
@@ -1,3 +1,3 @@
-build:
-  docker:
-    web: Dockerfile
+build:
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)